File: service/java/src/main/java/org/whispersystems/signalservice/internal/push/DeviceLimit.java
package org.whispersystems.signalservice.internal.push;

import com.fasterxml.jackson.annotation.JsonProperty;

public class DeviceLimit {

  @JsonProperty
  private int current;

  @JsonProperty
  private int max;

  public int getCurrent() {
    return current;
  }

  public int getMax() {
    return max;
  }
}

File: lib/libsignal-service/src/main/java/org/whispersystems/signalservice/internal/push/DeviceLimit.kt
/*
 * Copyright 2025 Signal Messenger, LLC
 * SPDX-License-Identifier: AGPL-3.0-only
 */

package org.whispersystems.signalservice.internal.push

data class DeviceLimit(
  val current: Int = 0,
  val max: Int = 0
)
